src/components/study-pane.js

```diff
@@ -155,20 +145,20 @@ const StudyPane = ({
         }
     }, [network, filteredNominalVoltages, dispatch]);
 
-    function openVoltageLevelDiagram(vlId, substationId) {
+    function openVoltageLevelDiagram(vlId) {
```

Comment:
Could be openVoltageLevelDiagramFromResults

Reply:
This is not my code, I don't think I should modify it in this refactoring. Maybe in another US/TS